packages/ast-spec/src/element/TSEnumMember/spec.ts

```diff
@@ -19,12 +19,18 @@ interface TSEnumMemberBase extends BaseNode {
  * this should only really happen in semantically invalid code (errors 1164 and 2452)
  *
  * VALID:
+ * @example
```

Comment:
This doesn't work. A JSDoc comment cannot have duplicate keys and here there are two `@example` keys. I suggest just using plain descriptions instead.

Reply:
Huh, TIL - thanks! I just reduced it down to the one `@example` to keep the change minimal.